scripts/genproto.sh

```diff
@@ -16,9 +16,14 @@ if ! [[ $(protoc --version) =~ "3.0.0" ]]; then
 fi
 
 export PATH=.:${PATH}
+cd $GOPATH/src/github.com/coreos/rkt
```

Comment:
not sure I understand correctly. Is this still runnable by a user with `GOPATH` unset? if so, this is fine with me.